src/utils/DateUtils.ts

```diff
@@ -1,24 +1,125 @@
 import { Options } from '../misc/Options';
 import { Utils } from './Utils';
 import { l } from '../strings/Strings';
-import { TimeSpan } from './TimeSpanUtils';
-import * as Globalize from 'globalize';
 import * as _ from 'underscore';
-
+import * as moment from 'moment';
+/**
+ * The 'DateUtils' class renders dates and times using momentJS using the right culture languages and formats.
+ *
+ * This class can transform strings, numbers and date objects into a standardized ISO_8601 Date.
+ *
+ */
 export interface IDateToStringOptions {
+  /**
+   * Specifies whether the date and time are the current ones.
+   */
   now?: Date;
+
```

Comment:
In general, I would use the active voice whenever possible (i.e., "Specifies whether to display the date" rather than "Specifies whether the date should be displayed").

Also, use back-ticks (`) rather than single quotes (') when you want to render short code blocks on the generated docs site (e.g., `Today` rather than 'Today').

Content-wise, I'm not quite sure we should talk about the date being otherwise displayed "normally" for any of the options in the `IDateToStringOptions` interface. If I understand correctly, I think such a statement would be slightly misleading. Moreover, it is not entirely clear to me what "normally" means in this context (I'm guessing it's the number of MS since Jan 1 1970 UTC, but it's only a guess).

All things considered, I would change this to:

> Specifies whether to convert the Date object to the localized version of `Today`, `Yesterday`, or `Tomorrow`, if able.
>
> **Examples:**
>
> If `useTodayYesterdayAndTomorrow` is `true`, and [`now`]{@link IDateToStringOptions.now} contains a Date object equivalent to `March 8, 2017`then:
> - If the Date object to convert contains a value equivalent to `March 8, 2017`, the resulting string is the localized version of `Today`.
> - If the Date object to convert contains a value equilavent to `March 7, 2017`, the resulting string is the localized version of `Yesterday`.
> - If the Date object to convert contains a value equivalent to `March 9, 2017`, the resulting string is the localized version of `Tomorrow`.
>
> Default value is `true`.